pelican/readers.py

```diff
@@ -268,7 +268,11 @@ def _parse_metadata(self, meta):
                 formatted_values = "\n".join(value)
                 # reset the markdown instance to clear any state
                 self._md.reset()
+                # process field as markdown, but *don't* extract md metadata
+                preprocessors = self._md.preprocessors
+                self._md.preprocessors.pop('meta', None)
                 formatted = self._md.convert(formatted_values)
+                self._md.preprocessors = preprocessors
```

Comment:
This won't do what you intend it to do. I assume you wanted to 'cache' the Markdown preprocessors instance containing `meta` and reverse it afterwards. But you're mutating `self._md.preprocessors`, so the 'cached' `preprocessors` will be the same object regardless (won't contain `meta`).

In any case, I don't think this is needed. A new Markdown instance is created for every read and the one in `self._md` isn't reused after this.